Bugfix: Misc & Spotify improvements

fatfingers23 opened the pull request:
Misc
- Resolved the DID mix up. It had a `OR ID` so was always true and returning the top user in the db causing that to be the only PDS that tracks got sent to
- Standardized time across the board to `.UTC()` because `time.now()` was doing local time
- Commented out or changed the wording for anywhere printing credentials so it wont be in the logs since there has been some chatter about a public instance 

Spotify
- Record refreshtoken on login
- Load and unload users each loop and at the start checks if the token is expired, if it is it refreshes it

Have tested a bit but plan on leaving it on over the weekend to see if anything else comes up

espeon reviewed (commented):
looks fine to me so far, lmk when to merge it once you've tested it 🫡

fatfingers23 commented:
Alright feeling a bit better with testing

1. Updated to the new lexicons for plays
2. Added dedicated loggers with prefixes to help while troubleshooting
3. Fixed spotify hangup son token refreshes was a mutex hangup. In the load it was locking while inside when trying to update it would hang on a lock. Should hopefully have no mutex blocking now

fatfingers23 commented:
Resolved Music Brainz fields being rendered when they are empty 

Used this to test 
https://open.spotify.com/track/7MtlureL5VXmDnyoUeBxsd?si=1236dcbdb3c84db5
